batch_wind_tunnels.py

```diff
@@ -0,0 +1,65 @@
+"""Submits simulations to the API."""
+import numpy as np
+
+from lib import models
+from lib import scenarios
+
+import inductiva
+
+OBJECT_PATH = "metaballs/metaball.obj"
+WORKING_DIR = None
+
+FLOW_VELOCITY_RANGE_X = [20, 30]
+FLOW_VELOCITY_RANGE_Y = [0, 0]
+FLOW_VELOCITY_RANGE_Z = [0, 0]
+NUM_SIMULATIONS_PER_OBJECT = 1
+
+DOMAIN_X = [-5, 20]
+DOMAIN_Y = [-5, 5]
+DOMAIN_Z = [0, 8]
+NUM_ITERATIONS = 100
+RESOLUTION = 2
+
+MACHINE_TYPE = "c2-standard-16"
+NUM_MACHINES = 1
+DISK_SIZE_GB = 70
+
+
+def run_simulation(machine_group):
+    flow_velocity = [
```

Comment:
To be fair, I think the batch job would make more sense with the notion of selecting different parameters. So, this flow_velocity would be set down below on the main function, while the rest is kept intact.

Reply:
I don't agree, since the API does not natively support batch job submissions I think that this should be set here in the function that submits a single task. Otherwise we would have to do something like:

```python
velocities = # some [NUM_SIMULATIONS, 3] array

for velocity in velocities:
    run_simulation(velocity,...)
```

This can be unnecessarily wasteful in terms of memory for very large jobs.

Reply:
Or...

```python
for _ in range(NUM_SIMULATIONS_PER_OBJECT):
     flow_velocity = [
        np.random.uniform(FLOW_VELOCITY_RANGE_X[0], FLOW_VELOCITY_RANGE_X[1]),
        np.random.uniform(FLOW_VELOCITY_RANGE_Y[0], FLOW_VELOCITY_RANGE_Y[1]),
        np.random.uniform(FLOW_VELOCITY_RANGE_Y[0], FLOW_VELOCITY_RANGE_Z[1]),
    ]
    run_simulation(flow_velocity, machine_group)
```

Reply:
Just to give a notion we are iterating over something.